### PR TITLE
2092 rebuild test configurations

### PR DESCRIPTION
**Pull request by mssalvatore**
# What does this PR do?

Add new test configurations based on AgentConfiguration

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Have you successfully tested your changes locally? Elaborate:
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    Credentials(None, Password("Xk8VDTsC")),
+)
+
+
+def _add_exploiters(agent_configuration: AgentConfiguration) -> AgentConfiguration:
+    brute_force = [
+        PluginConfiguration(name="HadoopExploiter", options={}),
+        PluginConfiguration(name="Log4ShellExploiter", options={}),
+        PluginConfiguration(name="MSSQLExploiter", options={}),
+        PluginConfiguration(name="SmbExploiter", options={}),
```
[the remark this answers is not in the document]
Comment (shreyamalviya) [reply envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py:26]: And for credentials in other brute-force exploiters

[the remark this answers is not in the document]
Comment (shreyamalviya) [reply envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py]: Where do these credentials come from?
image

[the remark this answers is not in the document]
Comment (mssalvatore) [reply envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py]: The `AutomatedMaster` mixes them in.

**Review by ilija-lazoroski — COMMENTED** [review]
Can we fix vulture warnings ?
image

**Comment by shreyamalviya** [comment]
@ilija-lazoroski they'll go away when we modify the test code to use the new configurations (WIP).

**Review by shreyamalviya — APPROVED** [review]

**Review by ilija-lazoroski — APPROVED** [review]

**Pull request merged**